src/webrtc/call.ts

```diff
@@ -1997,7 +1997,10 @@ export class MatrixCall extends EventEmitter {
         if (this.opponentDeviceId) {
             return this.client.sendToDevice(eventType, {
                 [this.invitee || this.getOpponentMember().userId]: {
-                    [this.opponentDeviceId]: realContent,
+                    [this.opponentDeviceId]: {
+                        ...realContent,
+                        device_id: this.client.deviceId,
```

Comment:
I am wondering if the js-sdk wouldn't allow you to get the deviceId from the `MatrixEvent`, so that you don't have to send it

Reply:
Indeed, you should be able to find out through the sender key of the olm decryption (unsure if/how the js-sdk exposes that though), which has the added benefit that it can't be spoofed. See https://github.com/matrix-org/matrix-doc/pull/3401#discussion_r805893936